runtime/parachains/src/dmp.rs

```diff
@@ -68,24 +112,47 @@ impl fmt::Debug for ProcessedDownwardMessagesAcceptanceErr {
 	}
 }
 
+/// To readjust the memory footprint when sending or receiving messages we will split
+/// the queue in pages of `QUEUE_PAGE_CAPACITY` capacity. Tuning this constant allows
+/// to control how we trade off the overhead per stored message vs memory footprint of individual
+/// messages read. The pages are part of ring buffer per para and we keep track of the head and tail page index.
+///
+/// TODO(maybe) - make these configuration parameters?
+///
+/// Defines the queue page capacity.
+pub const QUEUE_PAGE_CAPACITY: u32 = 32;
+/// Defines the maximum amount of pages returned by `dmq_contents`.
+pub const MAX_PAGES_PER_QUERY: u32 = 2;
```

Comment:
Yeah, fixed that, also added some justifications for the value. I don't really think it requires a migration, as this will apply only to new pages of messages that are queued. All messages stored before changing the value can be still be retrieved.